drivers/common/opentimers.c

```diff
@@ -461,9 +466,10 @@ void opentimers_timer_callback(void){
     }
     
     // 4. reschedule the timer
-    opentimers_vars.currentTimeout = opentimers_vars.timersBuf[idToSchedule].currentCompareValue;
-    opentimers_vars.lastCompare[opentimers_vars.index] = opentimers_vars.currentTimeout;
-    opentimers_vars.index = (opentimers_vars.index+1)&0x0F;
-    sctimer_setCompare(opentimers_vars.currentTimeout);
-    opentimers_vars.running        = TRUE;
-}
\ No newline at end of file
+	if (TRUE==opentimers_vars.running){
```

Comment:
please make sure the indention is correct and the following code.

Reply:
Could you swipe the condition by putting the variable name on the left and the possible value on the right?